fmriprep/workflows/fieldmap/phdiff.py

```diff
@@ -131,6 +132,11 @@ def _pick1st(inlist):
 # Helper functions
 # ------------------------------------------------------
 
+@due.dcite(
+    Doi('10.1006/nimg.2001.1054'),
+    description="Conversion of the input phase-difference map into a fieldmap in Hz",
+    tags=['edu']
+)
```

Comment:
What is specific problem or Could you please share some easy way to reproduce the problem? (got lost a bit in that circle-ci setup to actually run the test(s))